backend/backend.c

```diff
@@ -113,7 +113,7 @@ struct wlr_backend *wlr_backend_autocreate(struct wl_display *display) {
 		struct wlr_backend *wl_backend = attempt_wl_backend(display);
 		if (wl_backend) {
 			wlr_multi_backend_add(backend, wl_backend);
-			return backend;
+			//return backend;
```

Comment:
wat

Reply:
This is the hack that spawns both a Wayland and a X11 backend. I should have added a comment to make it clear that it's not there to stay. That's the reason why this PR is marked as "DO NOT MERGE".